2.1/build/rpmbuild/SOURCES/cc-oci-runtime.spec

```diff
@@ -0,0 +1,131 @@
+%undefine _missing_build_ids_terminate_build
+Name:      cc-oci-runtime
+Version:   2.1.0.rc.6
+Release:   0
+Source0:   %{name}-%{version}.tar.gz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Summary  : No detailed summary available
+Group    : Development/Tools
+License  : Apache-2.0 GPL-2.0
+Requires: cc-oci-runtime-bin
+Requires: cc-oci-runtime-config
+Requires: cc-oci-runtime-data
+Requires:  clear-containers-image
+Requires:  clear-containers-selinux
+Requires:  qemu-lite
+Requires:  linux-container
+Patch1: 0001-Define-GOPATH-if-there-is-none.patch
+
+%description
+.. contents::
+.. sectnum::
+``cc-oci-runtime``
+===================
+Overview
+--------
+
+%package bin
+Summary: bin components for the cc-oci-runtime package.
+Group: Binaries
+Requires: cc-oci-runtime-data
+Requires: cc-oci-runtime-config
+
+%description bin
+bin components for the cc-oci-runtime package.
+
+%package config
+Summary: config components for the cc-oci-runtime package.
+Group: Default
+
+%description config
+config components for the cc-oci-runtime package.
+
+%package data
+Summary: data components for the cc-oci-runtime package.
+Group: Data
+
+%description data
+data components for the cc-oci-runtime package.
+
+%package extras
+Summary: extras components for the cc-oci-runtime package.
+Group: Default
+
+%description extras
+extras components for the cc-oci-runtime package.
+
+%prep
+%setup -q -n cc-oci-runtime-2.1.0-rc.6
+%patch1 -p1
+
+%build
+export LANG=C
+sh ./autogen.sh --disable-static --enable-valgrind \
+--disable-cppcheck \
+--disable-tests \
+--disable-valgrind-memcheck \
+--disable-functional-tests \
+--disable-docker-tests \
+--with-cc-kernel=/usr/share/clear-containers/vmlinux.container \
+--with-cc-image=/usr/share/clear-containers/clear-containers.img \
+--with-cc-image-systemdsystemunitdir=/usr/lib/systemd/system \
+--enable-autogopath
```

Comment:
This only works if you apply the patch that is submitted here:

https://github.com/01org/cc-oci-runtime/pull/599